tests/test-plugin-with-errors/test-plugin-with-errors.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
@vishalkakadiya can we move both plugins to the following directory `tests/testdata/plugins/...`. Theres other testdata being added and this will help to organise and separate those from the actual tests.

Reply:
Updated this now. Thank you!